instance/instance.go

```diff
@@ -140,20 +165,17 @@ func (h *Instance) ExactImages(images []flux.ImageID) (ImageMap, error) {
 
 // Checks whether the given image exists in the repository.
 // Return true if exist, false otherwise
-func (h *Instance) imageExists(image flux.ImageID) (exists bool, err error) {
-	// Get a list of images
-	images, err := h.registry.GetRepository(image.Repository())
+func (h *Instance) imageExists(image flux.ImageID) (bool, error) {
+	img, err := registry.ParseImage(string(image), nil)
```

Comment:
In fact, no, I can't use it. Because one of the bug reports was that it didn't error when the tag/image was malformed. It was because it never errors in the ImageID stuff.

So I'm keeping this as is, because it is safer.